### ISSUE TITLE
Optional `phone` field for Create Conversation `customer` object

### ISSUE THREAD

**Issue by bob983** [helpscout/helpscout-api-php#179]
When specifying a customer in Create Conversation endpoint, it's possible to add several optional fields like `firstName`, `lastName`, `orginatiozation`, etc.

The Mailbox API started supported `phone` field recently as well:

```json
{
...
  "customer" : {
    "email" : "bear@acme.com",
    "phone" : 800 123 123"
  }
}

```

**Comment by bkuhl** [comment]
Fortunately we are already [extract the entire `Customer`](https://github.com/helpscout/helpscout-api-php/blob/master/src/Conversations/Conversation.php#L322-L324) so we're covered there.  The only piece missing is the `phone`, which the #193 addresses.

**Comment by bkuhl** [comment]
This was included in [v2.4.0](https://github.com/helpscout/helpscout-api-php/releases/tag/2.4.0)
